src/colors.yaml

```diff
@@ -56,6 +59,10 @@ ui:
       dimmingOverlay: [0,0,0,0.4]
       tappable: [56,56,15,0.09]
       tappableInverted: [255,246,196,0.27]
+      fieldBG: [0,0,0,0.09]
```

Comment:
Yea, that makes more sense. Just have to think of what to rename these
colors of they're going in the base palette...